packages/kit/test/apps/basics/src/routes/selection/_tests.js

```diff
@@ -0,0 +1,34 @@
+import * as assert from 'uvu/assert';
+
+/** @type {import('test').TestMaker} */
+export default function (test) {
+	test('reset selection', '/selection/a', async ({ page, clicknav }) => {
+		await page.waitForTimeout(50);
+		assert.equal(
+			await page.evaluate(() => {
+				const range = document.createRange();
+				range.selectNodeContents(document.body);
+				const selection = window.getSelection();
+				if (selection) {
+					selection.removeAllRanges();
+					selection.addRange(range);
+					return selection.rangeCount;
+				}
+				return 0;
+			}),
+			1
+		);
+		await clicknav('[href="/selection/b"]');
+		await page.waitForTimeout(50);
```

Comment:
And the timeout here too.

Reply:
Test was inconsistent without that timeout, on my setup at least. Maybe someone can run tests to confirm.

Reply:
Can't reproduce inconsistencies so waitForTimeout calls was removed. By the way, is there a docker file or something for contributors, to build/test project?

Reply:
> By the way, is there a docker file or something for contributors, to build/test project?

There isn't one that I know of, but I don't have much issues with local test inconsistencies in the past. Playwright ships its own chromium browser IIRC so that should eliminate most of the cross-machine quirks.